app/assets/stylesheets/pages/_new_booking.scss

```diff
@@ -0,0 +1,4 @@
+#new_booking .form-select{
+  background: none !important;
```

Comment:
!important is needed to override bootstrap default styles. In this case I wanted to remove the little drop-down arrows on the date/time fields because they were redundant. 

I had checked with a TA once, !important is the correct way to do it.